apps/climatemappedafrica/src/payload/blocks/Hero.js

```diff
@@ -0,0 +1,73 @@
+import LocationSelect, { validateLocation } from "../fields/LocationSelect";
+import richText from "../fields/richText";
+
+const Hero = {
+  slug: "hero",
+  imageURL: "/images/cms/blocks/hero.png",
+  imageAltText: "Used in homepage",
+  labels: {
+    singular: "Hero",
+    plural: "Hero",
+  },
+  fields: [
+    richText({
+      name: "title",
+      required: true,
+      label: "Title",
+      localized: true,
+    }),
+    richText({
+      name: "subtitle",
+      required: true,
+      label: "Sub Title",
```

Comment:
```suggestion
      label: "Subtitle",
```

Or just remove `label` if it's the same as `name`.